blank-canvas-blocks/functions.php

```diff
@@ -88,3 +88,8 @@ function blank_canvas_blocks_fonts_url() {
 if ( ! is_child_theme() ) {
 	require get_template_directory() . '/inc/block-patterns.php';
 }
+
+/**
+ * Add Global Styles Variables
+ */
+require get_template_directory() . '/inc/global-styles-variables.php';
```

Comment:
I would be keen for this (and related) changes (moving foreground/background variable consumption from custom to generated variables) into trunk sooner than later (even without the color changing bits).  Definitely a good change.

Reply:
Yeah I agree. I'll make a PR for this.